site/topics/conditions/conditions.rst

```diff
@@ -26,6 +26,89 @@ Status Flags
 Flags Register
 ==============
 
+.. figure:: status_flags_observing_alu.png
+    :width: 500 px
+    :align: center
+
+    Status flag signals connected to an ALU. The output of the ALU would typically be sent back to the data bus, but
+    here the output is ignored except for analysing the output's status for the purpose of determining the status flags.
+
+
+* The goal is to know if the last arithmatic operation performed caused any of the status signals to change
+* However, the adder is always outputting a value based on the contents of registers A and B
+
+    * This is true, even when addition/subtraction is not intentionally being performed
+
+
+* For example, consider the steps the system performs when calculating ``5 - 5``
+
+    * Load ``5``` into A
+    * Load ``-5`` into B
+    * Set the subtraction signal
```

Comment:
Double negative here, this would be computing `5 - -5`

Reply:
good catch